src/components/Sum.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Sum = ({ a, b }) => (
```

Comment:
I think you could directly give default values here like `{ a = 0, b = 0 }` but your way still works.